examples/opf/experiments/anomaly/spatial/10field_many_balanced/UNDER_DEVELOPMENT

```diff
@@ -34,7 +34,7 @@ from nupic.frameworks.opf.expdescriptionhelpers import (
   applyValueGettersToContainer,
```

Comment:
There are a bunch of "UNDER_DEVELOPMENT" examples. I'll remove them in another PR.